src/openforms/registrations/base.py

```diff
@@ -60,3 +60,10 @@ def update_payment_status(self, submission: "Submission"):
 
     def get_label(self):
         return self.verbose_name
+
+    @abstractmethod
+    def test_config(self):
```

Comment:
The return type should be set on the base method. I used bool as example but, read the plugin comments first.

```suggestion
    def test_config(self) -> bool:
```